Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -92,8 +93,12 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
                 // Generic profile based on scatter color
                 result = TransmissionKernel(s, invScattering) * lightIntensity * E * transmission.GetScale();
 #else // USE_HUMAN_SKIN_PROFILE
-                // Profile specific to human skin
-                result = T(s) * lightIntensity * E * transmission.GetScale();
+                if(o_transmission_mode == TransmissionMode::ThinObject)
+                    // Profile specific to human skin
+                    result = T(s) * lightIntensity * E * transmission.GetScale();
+                else
+                    // Use Burley transmission in skin material
+                    result = TransmissionKernel(s, invScattering) * lightIntensity * E * transmission.GetScale();                
```

Comment:
Bump. @antonmic

Reply:
I think there's some confusion? The previous code was using Burley calculations if it was NOT using the USE_HUMAN_SKIN_PROFILE macro. Perhaps this was an error?
I agree with the sentiment of getting rid of USE_HUMAN_SKIN_PROFILE in favor of having it be part of the transmission option, but then the code needs to be changed, since previously TransmissionMode::ThinObject would default to Burley if the USE_HUMAN_SKIN_PROFILE macro wasn't specified, whereas if we remove the macro in the new code, TransmissionMode::ThinObject would result to the // Profile specific to human skin.
So maybe we don't need a Burley option? Maybe we just make Burley the default for ThinObject and make a new profile called TransmissionMode::Skin ? This would make it confusing for users who want to use Burley for skin though, as seems to be the case here.
Thoughts? What would the most straighforward set of options be if we designed this from scratch?
The material could has TransmissionMode::Thick/Thin/Skin, but when Skin is enabled we could have an additional checkbox that appears that says "use Burley for Skin"?
@thefranke You mention the in the review that this helps for ear transmission, a case where the Jimenez model didn't fair well? How would you apply this to the model, i.e. say I want the skin profile for the face because it works better but then want Burley for the ears? Would you set two different materials for the ears or would the difference need to be controlled with a texture?

Reply:
@antonmic You mean @327926791?

Reply:
@antonmic previously the problem is when we use skin material type, we can't select the Burley transmission method. So this change is to add an option to use Burley function. It will be confusing if we had an transmission mode name "Skin" when using a skin material type, also "skin" should be a subclass under "thin object".
Now, "thick object" transmission mode is using a simple lambert attenuation, "thin object" transmission mode is using two kinds of transmission kernel, Burley and Gaussian, to approximate the diffuse profile for human skin and non-human-skin material.
If we re-design the code to be more straightforward, we would suggest using characteristic of each algorithm or name of the algorithm as the transmission mode name, instead of thick/thin/skin. Something Like "Beer-Lambert Based Thick Object"/"Gaussian Based Thin Object"/"Burley Based Thin Object".

Reply:
How about:
- Thick Object (Beer-Lambert)
- Thin or Skin Object (Burley)
- Skin Object (Guassian)
Emphasizing the use cases will help artists, and we'd still have the algorithms in parenthesis for graphics engineers
(This is not a blocking issue btw, you can ship this PR and address this issue in a follow up PR)

Reply:
I felt there are highly similar between skin material and enhance PBR material, sharing most common ground features from enhance PBR including subsurface and transmission mode, but with different of some targeted skin feature added to skin material.  My thoughts if re-design from ground up should be done by single enhance PBR to create skin material. (may be there are user control skin features  options ported to enhance PBR to make it up). That's my thoughts.

btw, we (+@327926791) have new features want to compliment Burley transmission PR, we want to submit together with this, this PR will withdraw from now and with features later.

Reply:
Enhanced PBR is designed as a "everything altogether" look-dev material type. It has all the features and is great for testing and look development, but has poor performance and shouldn't be considered for performant runtime usages. The goal for skin is to have a dedicated skin material that is specialized and optimized to render skin.